src/script/user/UserRepository.ts

```diff
@@ -347,23 +333,18 @@ export class UserRepository {
     });
   }
 
-  private isTeamTooLargeForBroadcast(): boolean {
-    const teamSizeIncludingSelf = this.teamUsers().length + 1;
-    return teamSizeIncludingSelf >= UserRepository.CONFIG.MAXIMUM_TEAM_SIZE_BROADCAST;
-  }
-
```

Comment:
Not necessary anymore. This was only used for limiting the availability updates.